test/algorithms/ASB07.jl

```diff
@@ -0,0 +1,42 @@
+@testset "ASB07 algorithm" begin
+
+    # example 1 from
+    # "Reachability analysis of linear systems with uncertain parameters and inputs"
+    # (Adapted from Reachability.jl)
+
+    # initial set
+    X0 = BallInf([1.0, 1.0], 0.1)
+
+    # linear ODE: x' = Ax
+    A = IntervalMatrix([-1.0 ± 0.05 -4.0 ± 0.05;
+                         4.0 ± 0.05 -1.0 ± 0.05])
+    P_lin = @ivp(x' = Ax, x(0) ∈ X0)
+
+    # TODO affine ODE: x' = Ax + Bu
+    # B = IntervalMatrix(hcat([1.0 ± 0.01; 1.0 ± 0.0]))
+    # u = ConstantInput(Interval(-0.05, 0.05))
+    # P_aff = @ivp(x' = Ax + Bu, x(0) ∈ X0)
+
+    f1(P) = solve(P, tspan=(0.0, 1.0), alg=ASB07(δ=0.04));
```

Comment:
for now i'd just keep the test that works (you can comment the other parts if that may be useful in the future), something like

```julia
sol = solve(P, tspan=(0.0, 1.0), alg=ASB07(δ=0.04))
    @test isa(sol.alg, ASB07)
    @test setrep(sol) <: Zonotope
    @test setrep(sol) == . . . . .
    @test dim(sol) == . . . .
# any other relevant test?
```